app/src/main/java/org/sagebase/crf/camera/MaskableSurfaceView.java

```diff
@@ -0,0 +1,82 @@
+/*
+ *    Copyright 2017 Sage Bionetworks
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package org.sagebase.crf.camera;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Path;
+import android.util.AttributeSet;
+import android.view.SurfaceView;
+
+/**
+ * Created by TheMDP on 10/30/17.
+ */
+
+public class MaskableSurfaceView extends SurfaceView {
```

Comment:
Is this basically a circle cropped view for the camera?

Reply:
Exactly, ill add some comments